packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1037,13 +1040,21 @@ describe("Runtime", () => {
 				});
 			});
 			it("process remote op with unrecognized type and 'Ignore' compat behavior", async () => {
-				const futureRuntimeMessage: ContainerRuntimeMessage = {
-					type: "FROM_THE_FUTURE" as ContainerMessageType,
+				const futureRuntimeMessage: RecentlyAddedContainerRuntimeMessageDetails &
+					Record<string, unknown> = {
+					type: "FROM_THE_FUTURE",
 					contents: "Hello",
 					compatDetails: { behavior: "Ignore" },
 				};
 
-				const packedOp: Partial<ISequencedDocumentMessage> = {
+				const packedOp: Omit<
```

Comment:
@jason-ha This doesn't feel worth it to me for tests.  I'm not as worried about playing fast and loose with types in a unit test since it's about driving the runtime a certain way and validating the results.  But I also don't mind if you do :)

Reply:
The part I liked was that I could see what things in contract the test was skipping. Ideally the processing wouldn't say those are needed if they aren't. Perhaps some refactor in future can better separate what matters when.